app/scripts/directives/footer-version.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/* global isCoreVersionCompatible:false */
+
+// Common directive for Footer Version
+angular.module('openhimConsoleApp')
+  .directive('footerVersion', function(Api, config) {
+    return {
+      template:'<span ng-class="footerVersionsCompatible ? \'\' : \'version-incompatible\'" ng-if="footerConsoleVersion && footerCoreVersion">' +
```

Comment:
would this ng-class not be similiar to the one being used for the about page?
` ng-class="{ 'version-incompatible': footerVersionsCompatible == false }"`